bullet_train/app/controllers/concerns/account/teams/controller_base.rb

```diff
@@ -110,13 +110,20 @@ def update
 
   # # DELETE /teams/1
   # # DELETE /teams/1.json
-  # def destroy
-  #   @team.destroy
-  #   respond_to do |format|
-  #     format.html { redirect_to account_teams_url, notice: 'Team was successfully destroyed.' }
-  #     format.json { head :no_content }
-  #   end
-  # end
+  def destroy
+    if current_user.teams.size == 1
```

Comment:
More of an aside about BT in general than a comment about this particular PR, but I wish that we didn't need this conditional. I understand why we do, given the current state of things. But it would be nice if there was a little better handling of the "you don't belong to any teams" case so that we didn't need to prevent this.

Reply:
@jagthedrummer I totally agree, and I didn't want to write it this way initially. At first I tried to make it a `before_action` callback, but it didn't seem right when I started to actually write it out.

I knew we had the following code in the code base which raises this `RemovingLastTeamAdminException` error, but I didn't flesh out how to do it with a team so I just pushed things as is. I’ll see if I can make it raise an error for Teams in a similar way.

https://github.com/bullet-train-co/bullet_train-core/blob/3aa6c0a76f70d846bb23b2746a6368f71eab4df1/bullet_train/app/controllers/concerns/account/memberships/controller_base.rb#L70-L84